client/components/sidebar.vue

```diff
@@ -7,17 +7,62 @@
         <v-btn v-if="!documentPublished && $currentUserId" color="success" :to="{name: 'publishDocument', params: {documentId}}"><translate>document-publish</translate></v-btn>
       </v-toolbar>
     </v-card>
-    <v-layout row ref="commentsList">
-      <v-card v-for="comment of documentComments" :key="comment._id" :style="{marginTop: `${comment.marginTop}px`}" ref="comments">
-        <v-card-text>
-          {{comment.body}}
-        </v-card-text>
-      </v-card>
+    <v-layout row wrap ref="commentsList">
+      <v-flex @click.stop="comment.showAddCommentForm = !comment.showAddCommentForm" xs12 v-for="comment of documentComments" :key="comment._id" :style="{marginTop: `${comment.marginTop}px`}">
+        <v-card hover :class="['sidebar__comment', {'elevation-10':comment.focus}]" :style="{'padding-top':`${commentCardPaddingTop}px`, 'padding-bottom':`${commentCardPaddingBottom}px`}" ref="comments">
+          <v-container style="padding:0px">
+            <comment :comment="comment"/>
+            <v-container style="padding-top:5px; padding-bottom:5px" v-show="comment.hasManyReplies && !comment.showAllReplies">
```

Comment:
Hm, why using inline style and not put into a CSS class? If the purpose is that they are dynamic, then they should just come from the code, not template.